tests.sh

```diff
@@ -60,7 +58,7 @@ assert "_clean; skip_if false; assert_raises true; assert_end;" \
 assert "_clean; skip_if bash -c 'exit 1'; assert_raises false; assert_end;" \
 "all 0 tests passed."
 # subshells and pipes can be used in skip as well (albeit escaped)
-assert "_clean; skip_if 'cat /etc/passwd | grep \$(echo \$USER)';
+assert "_clean; skip_if 'id | grep \$(echo \$USER)';
```

Comment:
This also works on Mac OS X, as the user isn't guaranteed to be in /etc/passwd on Mac OS.